app/scripts/views/mixins/password-mixin.js

```diff
@@ -39,17 +32,10 @@ define(function (require, exports, module) {
       try {
         var passwordField = this.$('.password');
         if (isVisible) {
-          passwordField.attr('type', 'text').attr('autocomplete', 'off')
-            .attr('autocorrect', 'off').attr('autocapitalize', 'off');
```

Comment:
Ah I see, so this should be something like this:

```js
        if (isVisible) {
          passwordField.attr('type', 'text').attr('autocomplete', 'off')
            .attr('autocorrect', 'off').attr('autocapitalize', 'off');
          this.logViewEvent('password.visible');
        } else {
          passwordField.attr('type', 'password');
          passwordField.removeAttr('autocomplete')		
               .removeAttr('autocorrect').removeAttr('autocapitalize');
          this.logViewEvent('password.hidden');
        }
```

Your new change is missing `.removeAttr('autocomplete')` as part of the function call.